src/main/scala/zio/prelude/GenFs.scala

```diff
@@ -24,15 +24,6 @@ object GenFs {
         Gen.causes(gen, Gen.throwable)
     }
 
-  /**
```

Comment:
I added this on my previous PR, but just realized there's already a `GenF` for `Option` in zio-test